skipchain/skipchain.go

```diff
@@ -1285,11 +1218,6 @@ func newSkipchainService(c *onet.Context) (onet.Service, error) {
 		s.GetSingleBlock, s.GetSingleBlockByIndex, s.GetAllSkipchains,
 		s.CreateLinkPrivate, s.Unlink, s.AddFollow, s.ListFollow,
 		s.DelFollow, s.Listlink))
-	s.ServiceProcessor.RegisterStatusReporter("Skipblock", s.db)
```

Comment:
Why is there no status reporter anymore?

Reply:
Merge damage. Fixing.